src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -19,7 +19,10 @@
     <None Remove="UI\Images\NodeStates\hidden-64px.png" />
     <None Remove="UI\Images\NodeStates\info-64px.png" />
     <None Remove="UI\Images\warning_16px.png" />
+    <None Remove="Views\GuidedTour\HtmlPages\nodePackage.html" />
     <None Remove="Views\Menu\TrustedPathView.xaml" />
+    <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
+    <None Remove="Views\GuidedTour\HtmlPages\Resources\emptyImage.png" />
```

Comment:
Is this image still used? I believe I remove it as well

Reply:
@QilongTang removed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12860/commits/ab9dac8beced65ebb949df6a3d59919273f40a45